Sources/SpeziBluetoothServices/Characteristics/PLXSpotCheckMeasurement.swift

```diff
@@ -0,0 +1,144 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import ByteCoding
+import Foundation
+import NIOCore
+import SpeziNumerics
+
+
+/// A PLX Spot-check Measurement, as defined by the Bluetooth specification.
+/// - See: https://www.bluetooth.com/specifications/specs/plxs-html/
```

Comment:
Maybe we can format this as a link like `[Pulse Oximeter Service](https://www.bluetooth.com/specifications/specs/plxs-html/)`